ports/xbyak/vcpkg.json

```
@@ -1,7 +1,17 @@
 {
   "name": "xbyak",
-  "version-string": "5.991",
+  "version-string": "6.00",
```

```suggestion
  "version": "6.00",
```
This might be served by [version](https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md#version).

---

The manifest spec won't accept it with the following error:
```
    $ (a manifest): 'version' text was not a relaxed version:
Error: String `6.00` must only contain dot-separated numeric values without leading zeroes.
```

---

Also, I think the "major" version number here just indicates a new feature was added, and isn't indicative of an api break.

---

Seems this port is not suitable for `version` here. So let's keep it as original.